Files/UserControls/StatusCenter.xaml.cs

```diff
@@ -14,79 +15,100 @@
 
 namespace Files.UserControls
 {
-    public sealed partial class StatusCenter : UserControl
+    public sealed partial class StatusCenter : UserControl, IStatusCenterActions
     {
-        public static ObservableCollection<StatusBanner> StatusBannersSource { get; set; } = new ObservableCollection<StatusBanner>();
+        #region Public Properties
+
+        public ObservableCollection<StatusBanner> StatusBannersSource { get; set; } = new ObservableCollection<StatusBanner>();
```

Comment:
No longer static!!

Reply:
@d2dyno1 Yes! Does this by any chance effect the displaying of posted banners across tabs? We want the Status Center to show all operations across all tabs.

Reply:
@duke7553 haven't thought about it, I'll check it tomorrow. You can check it now if you have time to do so

Reply:
@duke7553 I've reverted it back to static